components/badges/badge.jsx

```diff
@@ -0,0 +1,70 @@
+var React     = require('react'),
+    classnames  = require('classnames'),
+    ImageTag  = require('../imagetag.jsx');
+
+
+var getStatus = function(statusVal) {
+  if (statusVal === undefined) {
+    return '';
+  } else {
+    statusVal = statusVal.toLowerCase();
+  }
+
+  var statusClasses = classnames('fa', 'fa-fw', Badge.statusIcons[statusVal]);
+  var statusIcon = <i className={statusClasses} />;
+  var classNames = 'label label-default ' + statusVal;
```

Comment:
(nitpicking, can be done later)
should we make use of `classnames()` here as well?